runtime/main/src/main/java/org/qbicc/runtime/main/VMHelpers.java

```diff
@@ -221,6 +222,10 @@ public static void JLT_start0(void_ptr_unaryoperator_function_ptr runFuncPtr, vo
         // TODO once invokedynamic is working for lambda expressions use addr_of_function to fetch the correct function pointer
         //function_ptr<UnaryOperator<void_ptr>> runFuncPtr = addr_of_function(VMHelpers::threadWrapperNative);
 
+        if (Build.Target.isPThreads()) {
```

Comment:
Did you mean:
```suggestion
        if (! Build.Target.isPThreads()) {
```

Reply:
I did. I also though I had pushed the fix :D